Index: 101.py
===================================================================
diff --git a/101.py b/101.py
deleted file mode 100644
--- a/101.py	
+++ /dev/null	
@@ -1,1 +0,0 @@
-#lets begin!
\ No newline at end of file
Index: 205.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/205.py b/205.py
new file mode 100644
--- /dev/null	
+++ b/205.py	
@@ -0,0 +1,21 @@
+list = [1,2,1,5,6,1,1,5,'a','a','b',5]
+d = {}
+
+for i in list:
+    if i not in d:
+        d[i] = 1
+    else:
+        d[i] += 1
+print(d)
+
+#or
+
+result = {}
+
+for i in list:
+    result.setdefault(i, 0)
+    result[i] += 1
+
+print(result)
+
+
Index: 204.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/204.py b/204.py
new file mode 100644
--- /dev/null	
+++ b/204.py	
@@ -0,0 +1,29 @@
+list = ['a', 1, 2, 5, 10, 'hello', 'b', 7, 10.2]
+# {"str" : 3, "int" : 5}
+
+str_counter = 0
+int_counter = 0
+d = {}
+
+for i in list:
+    if isinstance(i, str):
+        str_counter += 1
+    elif isinstance(i, int):
+        int_counter += 1
+
+d['str'] = str_counter
+d['int'] = int_counter
+
+print(d)
+
+# or
+
+result = {'str': 0, 'int': 0}
+
+for i in list:
+    if isinstance(i, str):
+        result['str'] += 1
+    elif isinstance(i, int):
+        result['int'] += 1
+
+print(result)
\ No newline at end of file
Index: 207.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/207.py b/207.py
new file mode 100644
--- /dev/null	
+++ b/207.py	
@@ -0,0 +1,16 @@
+d = {'g1': [1,100,5,98,72,101], 'g2': [5,71,49,63]}
+# result = {'g1': 101, 'g2': 71}
+
+for i in d:
+    d[i] = max(d[i])
+
+print(d)
+
+#or
+
+result = {}
+
+for k, v in d.items():
+    result.setdefault(k, max(v))
+
+print(result)
Index: 206.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/206.py b/206.py
new file mode 100644
--- /dev/null	
+++ b/206.py	
@@ -0,0 +1,10 @@
+s = 'kn176egw@1!3938!!!!@##jfdnhkguh???!?!?e738r'
+# {'?': 5, '!': 7, '@':2, '#' : 2}
+
+d = {'?': 0, '!': 0, '@': 0, '#': 0}
+
+for i in s:
+    if i in list(d.keys()):
+        d[i] += 1
+
+print(d)
\ No newline at end of file
