Index: 102.py
===================================================================
diff --git a/102.py b/102.py
deleted file mode 100644
--- a/102.py	(revision 6c3b0daeb44082c467731457607428f47503bc23)
+++ /dev/null	(revision 6c3b0daeb44082c467731457607428f47503bc23)
@@ -1,22 +0,0 @@
-'''
-this code receives a string and counts its number of uppercase and lowercase characters.
-'''
-
-upper_counter = 0
-lower_counter = 0
-non_letter_counter = 0
-
-s = input("input string: ")
-
-for i in s:
-    if i.isupper():
-        upper_counter += 1
-    elif i.islower():
-        lower_counter += 1
-    else:
-        non_letter_counter += 1
-
-print(f"number of uppercase letters: {upper_counter}")
-print(f"number of lowercase letters: {lower_counter}")
-print(f"number of non letters: {non_letter_counter}")
-
Index: 104.py
===================================================================
diff --git a/104.py b/104.py
deleted file mode 100644
--- a/104.py	(revision 6c3b0daeb44082c467731457607428f47503bc23)
+++ /dev/null	(revision 6c3b0daeb44082c467731457607428f47503bc23)
@@ -1,9 +0,0 @@
-'''
-this code receives a string and reverses it.
-'''
-
-input_string = input("input string: ")
-
-reversed_string = input_string[ : : -1]
-
-print(reversed_string)
Index: 101.py
===================================================================
diff --git a/101.py b/101.py
deleted file mode 100644
--- a/101.py	(revision 6c3b0daeb44082c467731457607428f47503bc23)
+++ /dev/null	(revision 6c3b0daeb44082c467731457607428f47503bc23)
@@ -1,14 +0,0 @@
-#lets begin!
-'''
-this code receives a string and prints its number of digits.
-'''
-
-counter = 0
-
-num = int(input("input number: "))
-
-while num != 0:
-    num = num // 10
-    counter += 1
-
-print(counter)
\ No newline at end of file
Index: 105.py
===================================================================
diff --git a/105.py b/105.py
deleted file mode 100644
--- a/105.py	(revision 6c3b0daeb44082c467731457607428f47503bc23)
+++ /dev/null	(revision 6c3b0daeb44082c467731457607428f47503bc23)
@@ -1,12 +0,0 @@
-'''
-this code prints half diamond using *s.
-'''
-n = int(input())
-
-#printing the upper part with the main line.
-for i in range(1, n + 1):
-    print(i * "*")
-
-#printing the lowerpart.
-for i in range(n - 1, 0, -1):
-    print(i * "*")
\ No newline at end of file
Index: 103.py
===================================================================
diff --git a/103.py b/103.py
deleted file mode 100644
--- a/103.py	(revision 6c3b0daeb44082c467731457607428f47503bc23)
+++ /dev/null	(revision 6c3b0daeb44082c467731457607428f47503bc23)
@@ -1,18 +0,0 @@
-'''
-this code will calculate max avg of 5 interaction of random made numbers.
-'''
-import random
-
-max_avg = 0
-
-for i in range(5):
-    first_num = random.randint(0,100)
-    second_num = random.randint(0,100)
-    interaction_avg = (first_num + second_num) / 2
-
-    if interaction_avg > max_avg:
-        max_avg = interaction_avg
-
-    print(f"interaction {i}: first_num: {first_num}, second_num: {second_num}")
-
-print("max average is: ", max_avg)
\ No newline at end of file
